common/lib/xmodule/xmodule/capa_module.py

```diff
@@ -220,7 +220,11 @@ def non_editable_metadata_fields(self):
     @property
     def problem_types(self):
         """ Low-level problem type introspection for content libraries filtering by problem type """
-        tree = etree.XML(self.data)
+        try:
+            tree = etree.XML(self.data)
+        except etree.XMLSyntaxError:
+            log.warn('Error parsing problem types from xml for capa module {}'.format(self.display_name))
```

Comment:
Log at the error level.  I just confirmed with @jibsheet that ERROR level logs won't wake up devops.